test/e2e-tests.sh

```diff
@@ -70,6 +70,9 @@ go_test_e2e -timeout=10m ./test/conformance || failed=1
 # So that we can also identify failing E2E tests.
 go_test_e2e -timeout=20m ./test/e2e || failed=1
 
+# So that we can also identify failing E2E Scale tests.
+go_test_e2e -timeout=10m ./test/e2e/scale || failed=1
```

Comment:
You're running the scale tests twice (together with all tests i line 71, separate here), since all tests under `e2e` have the same tag (e2e). Is this intended?

Reply:
No, it is non-obvious that `/...` is appending to that test target...

Reply:
I moved this to `./test/scale` to avoid this and added back two invocations, one for `e2e/conformance` and another for `scale`